index.js

```diff
@@ -1,5 +1,5 @@
-function helloWorld() {
-  return "Hello, World"
-}
+import { addComma } from './utils/numericFormatters.js';
 
-module.exports = helloWorld;
+module.exports = {
+  addComma,
+};
```

Comment:
2 questions on syntax preference:
1. Do you prefer single quotes or double?
2. Do you prefer to include trailing commas or not?

Reply:
I could go either way on the quotes. For trailing commas, I kind of prefer them. However, if you prefer not having them, I'm good w/that